Sources/StreamChatUI/ChatMessageList/Attachments/Poll/PollAttachmentOptionListView/PollAttachmentOptionListItemView.swift

```diff
@@ -0,0 +1,170 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+/// The view that displays a poll option in the poll option list view.
+open class PollAttachmentOptionListItemView: _View, ThemeProvider {
+    public struct Content {
+        /// The option that this view represents.
+        public var option: PollOption
+        /// The poll that this option belongs.
+        public var poll: Poll
+        
+        public init(option: PollOption, poll: Poll) {
+            self.option = option
+            self.poll = poll
+        }
+
+        /// Whether the current option has been voted by the current user.
+        public var isVotedByCurrentUser: Bool {
+            poll.hasCurrentUserVoted(for: option)
+        }
+
+        /// The number of votes this option has.
+        public var voteCount: Int {
+            poll.voteCount(for: option)
+        }
+
+        /// The ratio of the votes of this option in comparison with the number of total votes.
+        public var voteRatio: Float {
+            poll.voteRatio(for: option)
+        }
+    }
+    
+    public var content: Content? {
+        didSet {
+            updateContentIfNeeded()
+        }
+    }
+
+    // MARK: - Action Handlers
+
+    /// A closure that is triggered whenever the option is tapped either from the button or the item itself.
+    public var onOptionTap: ((PollOption) -> Void)?
+
+    // MARK: - UI Components
+
+    /// A label which displays the name of the option.
+    open private(set) lazy var optionNameLabel = UILabel()
+        .withoutAutoresizingMaskConstraints
+        .withBidirectionalLanguagesSupport
```

Comment:
General observation that this is not needed. Created a general [ticket](https://stream-io.atlassian.net/browse/PBE-5577) for cleaning it up.

Reply:
Yeah I don't remember when was this introduced but in some views this was not being activated some how. Most likely some of our custom views... 🤔